### DISCUSSION TITLE
Not allowed to load local resource

### DISCUSSION THREAD

**Discussion by Catalendas** [alex8088/electron-vite#437] [Q&A] [answered]
hi!

This error occurs when I upload an image with the input file type and I want to preview this image:

![image](https://github.com/alex8088/electron-vite/assets/82763928/0cb79b2e-fd53-45d6-b375-b79eec5669ad)

When placing the image statically I get the same error.
Is there any solution?

"electron": "^29.1.0",
"electron-vite": "^2.1.0",

**Comment by alex8088** [comment]
Your screenshot looks confusing. Need more details.

**Reply by Catalendas** [reply]
![image](https://github.com/alex8088/electron-vite/assets/82763928/3e12f948-6496-42ef-b531-b2f8acd7ea09)

This is the image I'm uploading, it's in a local file and elctron gives me this error, saying it doesn't have permission for local files

**Reply by alex8088** [reply, marked as answer]
Try setting `webSecurity` to false

**Reply by Catalendas** [reply]
it worked, thank you
